### PR TITLE
Avoid embedding web-worker polyfill

### PR DESCRIPTION
**Pull request by moio**
### Summary

Fixes: #12511

**TL;DR**: Drop the web-worker polyfill. It's for ancient browsers and confuses tooling that works with performance captures.

**QA NOTE**: No user visible change expected, this is only to aid developers interpreting performance captures, eg. following https://github.com/rancherlabs/rancher-process/blob/main/docs/documentation/howtos/browser_performance_capture.md

### Technical notes summary
The BLOB code from web-worker is minified, and contains source maps that are then passed over by webpack - that creates a situation of minified code inside BLOB inside minified code which confuses some tools, eg.

https://github.com/jantimon/chrome-profile-sourcemap-resolver/pull/4

That could be fixed, but the only reason to do so would be supporting very old (pre-2013) browsers, so we are better off removing the polyfill altogether.

### Areas or cases that should be tested
Nothing in particular, exisiting tests should abundantly cover this.

### Areas which could experience regressions

None expected.

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

**Comment by moio** [comment]
End-to-end failures seem unrelated to me, but please advise if they are not.

**Comment by richard-cox** [comment]
Fallback came in via https://github.com/rancher/dashboard/pull/6563

**Review by richard-cox — COMMENTED** [review]
Think this is ok, but would like to confirm a prod build running remotely works (given https://github.com/rancher/dashboard/pull/6563). Just making one now (`richard-dev`).

If that's fine then we just need to wait for 2.10 branching (happening right now)

**Comment by richard-cox** [comment]
@moio same for this one
- [2.9.5](https://github.com/rancher/dashboard/issues/12515) 
- [2.10.1](https://github.com/rancher/dashboard/issues/12514)
- [2.11.0](https://github.com/rancher/dashboard/issues/12511) (this one)

**Review by richard-cox — APPROVED** [review]
Tested prod build, looks good.

I've bumped the tests again. Might need a rebase though (fair amount of dependency changes around release time)

**Comment by moio** [comment]
I force pushed and test ran again, the three failed ones are still failing, but I do not see how those failures can be related to the change.

Please merge when possible or let me know if anything needs to be done from my part, Thanks in advance

**Comment by richard-cox** [comment]
Required a couple of bumps, all passing now.
